.ci/ansible/tasks/install_deps.yml

```diff
@@ -9,6 +9,7 @@
       - libffi-dev
       - python3-dev
       - python3-pip
+      - python3-venv
```

Comment:
I'd say that this dependency is only needed at the Ansible controller machine, not for the remote Debian/Ubuntu instance

Reply:
Thanks @mdelapenya for clarifying the dependency here.